spec/features/jobs_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+RSpec.describe "Jobs" do
+  let(:client) { RenderAPI.client "test-api-key" }
+  let(:service_id) { "random-string" }
+
+  subject { client.jobs }
+
+  describe "create" do
+    before :each do
+      stub_request(
+        :post, "https://api.render.com/v1/services/#{service_id}/jobs"
+      ).to_return_json(
```

Comment:
Should I have used `with_json_body` here?

Reply:
Nope, you've got it correct as it is - `with_json_body` is for confirming a request that has been made **with a given JSON payload**, which isn't something I've been doing in existing specs, but these Webmock helpers are something I've written for other projects and copied into this repo.

An example for context:

```ruby
expect(
  a_request(:post, "https://api.render.com/v1/services/#{service_id}/jobs")
    .with_json_body { |json| json["startCommand"] == "whoami" }
).to have_been_made.once
```